.storybook/webpack.config.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
                 test: /\.tsx?$/,
                 loaders: [
                     'awesome-typescript-loader',
-                    'react-docgen-typescript-loader',
+                    // 'react-docgen-typescript-loader',
```

Comment:
@sedwards2009 found that disabling this speeds up the build no end. Is anybody using the functionality provided by [react-docgen-typescript-loader](https://github.com/strothj/react-docgen-typescript-loader)?